tests/conftest.py

```diff
@@ -948,6 +948,67 @@ def projects_dir() -> str:
     yield os.path.join(os.path.dirname(os.path.abspath(__file__)), "data")
 
 
+@pytest.fixture(scope="function")
+def project_builder(tmpdir: py.path.local) -> Callable[[str, List[str], List[str]], Project]:
+    """
+    Fixture that creates an Inmanta Project and returns the corresponding instance of the Project class.
+    """
+
+    def create_project(
+        main_cf_content: str,
+        copy_to_libs_dir: List[str] = [],
+        install_v2_modules: List[Tuple[str, bool]] = [],
+        python_package_source: Optional[str] = None,
+    ) -> Project:
+        """
+        :param main_cf_content: The content of main.cf file in the project.
+        :param copy_to_libs_dir: The V1 module directories that should be copied into the libs directory of the project.
+        :param install_v2_modules: The V2 module directories that should be installed in the developer venv.
+                                   The first element of the tuple indicates the path to the module directory. The second
+                                   element indicates whether the module should be installed in an editable way.
+        :param python_package_source: The python package repository that should be configured on the Inmanta project in order to
+                                      discover V2 modules.
+        """
+        project_dir = os.path.join(tmpdir, "project")
+        os.mkdir(project_dir)
+        project_yml_file = os.path.join(project_dir, "project.yml")
+        with open(project_yml_file, "w") as fd:
+            fd.write(
+                """
+name: test
+modulepath: libs
+downloadpath: libs
+install_mode: "release"
+repo:
+    - {type: git, url: https://github.com/inmanta/}
+"""
+            )
+            if python_package_source:
+                fd.write(f"    - {{type: package, url: {python_package_source} }}\n")
+
+        main_cf_file = os.path.join(project_dir, "main.cf")
+        with open(main_cf_file, "w") as fd:
+            fd.write(main_cf_content)
+
+        libs_dir = os.path.join(project_dir, "libs")
+        for mod_dir in copy_to_libs_dir:
+            shutil.copytree(mod_dir, os.path.join(libs_dir, os.path.basename(mod_dir)))
+
+        project = Project(path=project_dir)
+        project.use_virtual_env()
+        wheels_dir = os.path.join(tmpdir, "wheels")
+        for mod_dir, do_editable_install in install_v2_modules:
+            if do_editable_install:
+                project.install_in_compiler_venv(mod_dir, editable=do_editable_install)
+            else:
+                v2_package = ModuleTool().build(mod_dir, output_dir=wheels_dir)
+                project.install_in_compiler_venv(v2_package, editable=do_editable_install)
```

Comment:
Might it be cleaner to just delegate this to `ModuleTool().install`? This somewhat duplicates the logic of that method.